docs/TPMmodulePinout.md

```diff
@@ -0,0 +1,100 @@
+## TPM modules pinout
+This chapter contains a few different variants of the TPM module pinout. 
+This research was made to design a few variants of TPM modules with the most 
+popular pinouts. In this chapter below, the same pinouts of TPM modules were 
+grouped.
+Research is based on the best sellers positions form 
+[Newegg.com](https://www.newegg.com/) supplier.
+
+## 1. AsRock H170M PRO4 - LPC
```

Comment:
@piotrdanio Are you sure it is not comatible with the 2. pinout? I would say it is.

Reply:
ok